views/homepage.handlebars

```diff
@@ -105,12 +103,22 @@
           Click begin and then choose your favourite spotify track in a 1 V 1 battle or "Sputtle"
         </p>
       </div>
+        <div class="columns is-centered">
+            <div class="column is-align-items-flex-end">
+                <a class="button is-centered is-success is-large" id="beginMatch">Begin Match</a>
+            </div>
+        </div>
     </section>
 
     <section class="section media has-text-centered">
       <div class="container">
         <div class="columns is-centered is-8 is-variable">
-
+            {{#if loggedIn}}
+            {{#each matchup}}
+            {{> battles}}
+            {{/each}}
```

Comment:
if logged in the match up with populate and build cards out of the data! images fail but I can fix that soon